tools/specs/add-ons/Microsoft.Data.SqlClient.AlwaysEncrypted.AzureKeyVaultProvider.nuspec

```diff
@@ -25,31 +25,31 @@ Microsoft.Data.SqlClient.AlwaysEncrypted.AzureKeyStoreProvider.SqlColumnEncrypti
     <tags>sqlclient microsoft.data.sqlclient azurekeyvaultprovider akvprovider alwaysencrypted</tags>
     <dependencies>
       <group targetFramework="net462">
-        <dependency id="Microsoft.Data.SqlClient" version="5.1.2" />
+        <dependency id="Microsoft.Data.SqlClient" version="5.1.5" />
         <dependency id="Azure.Core" version="[1.35.0,2.0.0)" />
         <dependency id="System.Text.Encodings.Web" version="6.0.0" />
-        <dependency id="Azure.Security.KeyVault.Keys" version="[4.4.0,5.0.0)" />
+        <dependency id="Azure.Security.KeyVault.Keys" version="[4.5.0,5.0.0)" />
         <dependency id="Microsoft.Extensions.Caching.Memory" version="6.0.1" />
       </group>
       <group targetFramework="net8.0">
-        <dependency id="Microsoft.Data.SqlClient" version="5.1.2" />
+        <dependency id="Microsoft.Data.SqlClient" version="5.1.5" />
         <dependency id="Azure.Core" version="[1.35.0,2.0.0)" />
-        <dependency id="System.Text.Encodings.Web" version="6.0.0" />
-        <dependency id="Azure.Security.KeyVault.Keys" version="[4.4.0,5.0.0)" />
-        <dependency id="Microsoft.Extensions.Caching.Memory" version="6.0.1" />
+        <dependency id="System.Text.Encodings.Web" version="8.0.0" />
+        <dependency id="Azure.Security.KeyVault.Keys" version="[4.5.0,5.0.0)" />
+        <dependency id="Microsoft.Extensions.Caching.Memory" version="8.0.0" />
       </group>
       <group targetFramework="net6.0">
-        <dependency id="Microsoft.Data.SqlClient" version="5.1.2" />
+        <dependency id="Microsoft.Data.SqlClient" version="5.1.5" />
         <dependency id="Azure.Core" version="[1.35.0,2.0.0)" />
         <dependency id="System.Text.Encodings.Web" version="6.0.0" />
```

Comment:
System.text.encodings.web is part of the shared framework since .NET 6, so no explicit refrence is needed